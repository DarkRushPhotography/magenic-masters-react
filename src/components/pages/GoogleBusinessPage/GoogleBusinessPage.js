import React, { Component } from "react";

import PageHeader from "../../@shared/PageHeader/PageHeader";
import Page from "../../@shared/Page/Page";

class GoogleBusinessPage extends Component {

  render() {
    return (
      <Page>
        <PageHeader />
        <h1>Google Business</h1>
      </Page>
    );
  }
}

export default GoogleBusinessPage;
