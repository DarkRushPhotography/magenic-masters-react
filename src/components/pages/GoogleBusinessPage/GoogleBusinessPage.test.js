import React from "react";
import { shallow } from "enzyme";

import GoogleBusinessPage from "./GoogleBusinessPage";

describe("GoogleBusinessPage", () => {
  it("should render correctly", () => {
    const component = shallow(<GoogleBusinessPage />);

    expect(component).toMatchSnapshot();
  });
});