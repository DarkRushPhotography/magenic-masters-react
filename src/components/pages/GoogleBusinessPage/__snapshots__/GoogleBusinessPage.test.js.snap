// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleBusinessPage should render correctly 1`] = `
<Page>
  <withRouter(WithStyles(PageHeader)) />
  <h1>
    Google Business
  </h1>
</Page>
`;
